Answer 'How can I start using conda?' for FAQ doc (#86)
conda-incubator/conda-dot-org: learn/faq.md

A feedstock is a conda package repository.

## How can I start using conda?

> There are several ways to get started with [Conda](./faq.md/#what-is-conda). See the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for step by step instructions. You can [search for the packages](./faq.md/#how-can-i-search-for-packages) you require and also check out [conda channels](./faq.md/#what-is-a-conda-channel). In case you face any issues, feel free to [contact us](./faq.md/#where-can-i-find-help).

Reviewer: Conda is lowercase unless starting a sentence. I also tweaked a few sentence-structure things.

> There are several ways to get started with [Conda](./faq.md/#what-is-conda). See the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for step by step instructions. You can [search for the packages](./faq.md/#how-can-i-search-for-packages) you require and also check out [conda channels](./faq.md/#what-is-a-conda-channel). In case you face any issues, feel free to [contact us](./faq.md/#where-can-i-find-help).

Author: Noted, thank you for the review!

For more information, read [How to reach us](/community/#how-to-reach-us) on the [Community](/community) page.

## How can I start using conda?

> There are several ways to get conda on your computer. You can refer to the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for a step by step procedure. 

Reviewer: This is a good start. As a user of conda, do you think you would want more out of an answer like this? Maybe point them to channel or package resources? Actually, a lot of the other FAQ answers cover things like this, so maybe that's unnecessary. We could point to those questions, though. And to help with that, let's put this question right after "What is a feedstock?"

Author: Sure, yes. I noticed that we have many questions covering important things I would need as a new user hence didn't add much detail. I'll put this question under "What is feedstock?", see if I can add anything that won't overlap much with another answer and link answers wherever relevant. Thanks!

Author: Hi @kathatherine, please take a look at the changes I have made and let me know if this is what you were expecting. Thank you!

> There are several ways to get conda on your computer. You can refer to the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for a step by step procedure. 

Reviewer: ```suggestion
There are several ways to get started with conda. See the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for step by step instructions. 
```
